Kickstarter-iOS/Views/Controllers/RewardsCollectionViewController.swift

```diff
@@ -0,0 +1,123 @@
+import Foundation
+import Library
+import KsApi
+import Prelude
+
+final class RewardsCollectionViewController: UICollectionViewController {
+  private let dataSource = RewardsCollectionViewDataSource()
+  private let viewModel = RewardsCollectionViewModel()
+
+  // Hidden scroll view used for paging
+  private let hiddenPagingScrollView: UIScrollView = {
+    UIScrollView()
+      |> \.isPagingEnabled .~ true
+      |> \.isHidden .~ true
+  }()
+
+  private let layout: UICollectionViewFlowLayout = {
+    UICollectionViewFlowLayout()
+      |> \.minimumLineSpacing .~ Styles.grid(3)
+      |> \.sectionInset .~ .init(all: Styles.grid(6))
+      |> \.scrollDirection .~ .horizontal
+  }()
+
+  private var flowLayout: UICollectionViewFlowLayout? {
+    return self.collectionView.collectionViewLayout as? UICollectionViewFlowLayout
+  }
+
+  private let peekAmountInset = Styles.grid(3)
```

Comment:
Do we need a property for this? We're only accessing it twice from the same scope of `viewDidLayoutSubviews`?

Reply:
This is removed in the[ "peek" functionality PR](https://github.com/kickstarter/ios-oss/pull/665) so to prevent merge conflicts I'd rather just leave it here for now.